Index: src/main/resources/archetype-resources/src/main/java/App.java
===================================================================
diff --git a/src/main/resources/archetype-resources/src/main/java/App.java b/src/main/resources/archetype-resources/src/main/java/App.java
deleted file mode 100644
--- a/src/main/resources/archetype-resources/src/main/java/App.java	(revision 28dc7e3f85a17fd7317a21fa452c120aa1546a93)
+++ /dev/null	(revision 28dc7e3f85a17fd7317a21fa452c120aa1546a93)
@@ -1,13 +0,0 @@
-package $org.example;
-
-/**
- * Hello world!
- *
- */
-public class App 
-{
-    public static void main( String[] args )
-    {
-        System.out.println( "Hello World!" );
-    }
-}
Index: src/main/resources/archetype-resources/src/test/java/AppTest.java
===================================================================
diff --git a/src/main/resources/archetype-resources/src/test/java/AppTest.java b/src/main/resources/archetype-resources/src/test/java/AppTest.java
deleted file mode 100644
--- a/src/main/resources/archetype-resources/src/test/java/AppTest.java	(revision 28dc7e3f85a17fd7317a21fa452c120aa1546a93)
+++ /dev/null	(revision 28dc7e3f85a17fd7317a21fa452c120aa1546a93)
@@ -1,38 +0,0 @@
-package $org.example;
-
-import junit.framework.Test;
-import junit.framework.TestCase;
-import junit.framework.TestSuite;
-
-/**
- * Unit test for simple App.
- */
-public class AppTest 
-    extends TestCase
-{
-    /**
-     * Create the test case
-     *
-     * @param testName name of the test case
-     */
-    public AppTest( String testName )
-    {
-        super( testName );
-    }
-
-    /**
-     * @return the suite of tests being tested
-     */
-    public static Test suite()
-    {
-        return new TestSuite( AppTest.class );
-    }
-
-    /**
-     * Rigourous Test :-)
-     */
-    public void testApp()
-    {
-        assertTrue( true );
-    }
-}
